public/locales/en/translation.json

```diff
@@ -26,6 +28,7 @@
   "caretaker": "Caretaker",
   "city": "City",
   "voivodeship": "Voivodeship",
+  "amenities": "Amenities",
```

Comment:
amenities już były wcześniej dodane do translation en i pl, teraz są tam dwa razy- jedno jest wewnątrz "caretakerSearch", a drugie poza

Reply:
Racja, jest to zdublowane. Usunę duplikat